nuget/WindowsMobileCenterAnalytics.nuspec

```diff
@@ -17,17 +17,17 @@
   </metadata>
   <files>
 
-    <file src="nuget/Microsoft.Azure.Mobile.Analytics.targets"  target="build/uap10.0/Microsoft.Azure.Mobile.Analytics.targets"/>
+    <file src="nuget/Microsoft.Azure.Mobile.Analytics.targets"  target="build/uap10.0"/>
```

Comment:
I just removed the target file name to be consistent with the other nuspecs we have (if you only give the target directory it uses the source file name)